core/src/main/java/io/questdb/cairo/wal/WalEventReader.java

```diff
@@ -185,14 +185,25 @@ public WalEventCursor of(Path path, long segmentTxn) {
             // Check only lower short to match the version
             // Higher short can be used to make a forward compatible change
             // by adding more data at the footer of each record
-            final int formatVersion = Numbers.decodeLowShort(eventMem.getInt(WAL_FORMAT_OFFSET_32));
-            if (WALE_FORMAT_VERSION != formatVersion && WALE_MAT_VIEW_FORMAT_VERSION != formatVersion) {
+            final int version = eventMem.getInt(WAL_FORMAT_OFFSET_32);
+            final short formatVersion = Numbers.decodeLowShort(version);
+            if (formatVersion != WALE_FORMAT_VERSION && formatVersion != WALE_MAT_VIEW_FORMAT_VERSION) {
                 throw TableUtils.validationException(eventMem)
-                        .put("Wal events file version does not match runtime version [expected=")
+                        .put("WAL events file version does not match runtime version [expected=")
                         .put(WALE_FORMAT_VERSION).put(" or ").put(WALE_MAT_VIEW_FORMAT_VERSION)
                         .put(", actual=").put(formatVersion)
                         .put(']');
             }
+            final short messageVersion = Numbers.decodeHighShort(version);
```

Comment:
nit: I'd prefer the term minor version for that. The message and format versions mean the same thing to me.